### DISCUSSION TITLE
This commit does not belong to any branch on this repository, and may belong to a fork outside of the repository.

### DISCUSSION THREAD

**Discussion by vasild** [community/community#122781] [Repositories]
### Select Topic Area

Bug

### Body

This message: `This commit does not belong to any branch on this repository, and may belong to a fork outside of the repository` is displayed even for commits that belong to a branch in the repository, for example (all those commits belong to the `master` branch):

https://github.com/vim/vim/commit/8d8cb45756cb7e6fda17013b8347be3a11b29610

![vim](https://github.com/community/community/assets/266751/6de348fa-b116-475e-8653-84677c53f79d)

https://github.com/ElementsProject/lightning/commit/e38c828bf07f7bfc6fb29241e82b9e204f2d1e90

![ln](https://github.com/community/community/assets/266751/2b93b21b-b038-4d66-ac69-2403912f60ea)

https://github.com/bitcoin/bitcoin/commit/ef09f535b7b540d997f8c2bfa67b3386711bc8b4

![bitcoin](https://github.com/community/community/assets/266751/c2496324-d27e-43f0-bf74-9501d19c05d5)

**Comment by 1440000bytes** [comment]
I dont see those warnings when I open the commits.

**Comment by biztos** [comment]
I'm having exactly this problem, almost two years later.
